src/pudl/transform/ferc1.py

```diff
@@ -5608,7 +5638,12 @@ def calculation_components_xbrl_ferc1(**kwargs) -> pd.DataFrame:
             table_dimensions=table_dimensions_ferc1,
             dimensions=other_dimensions(),
         )
+        .pipe(
+            remove_non_totals_from_impiled_child_calculation_components,
+            dimensions=other_dimensions(),
```

Comment:
The `other_dimensions()` with no arguments seems weird. Like it's a constant that doesn't need to be a function, or like it ought to be `self.other_dimensions()` Would it be terrible to actually require the `table_names` argument to be stated explicitly rather than surreptitiously slipped in via a global variable if the function gets None?

Reply:
I just think it's clearer what kind of thing is happening and what the dependencies are if the table names argument is explicit -- even if it's just referring to the same global variable that's currently being injected within the function in the case that it gets no arguments.